proto/cosmos/staking/v1beta1/staking.proto

```diff
@@ -320,6 +320,13 @@ message Params {
     (amino.dont_omitempty) = true,
     (cosmos_proto.scalar)  = "cosmos.Dec"
   ];
+  // max_cons_pubkey_rotations is maximum number of consensus pubkey rotations
```

Comment:
this is within the unbonding period right?

Reply:
> this is within the unbonding period right?

No, overall.